xmtp_mls/src/groups/mod.rs

```diff
@@ -1002,6 +1002,19 @@ impl<ScopedClient: ScopedGroupClient> MlsGroup<ScopedClient> {
             })
     }
 
+    /// Find the `inbox_id` of the group member who is the peer of this dm
+    pub fn dm_inbox_id(&self) -> Result<String, GroupError> {
+        let conn = self.context().store().conn()?;
+        conn.find_group(self.group_id.clone())
```

Comment:
This would be tidier as something like
```rust
let group = conn.find_group(self.group_id.clone())?.ok_or(GroupError::GroupNotFound)?;
Ok(group.dm_inbox_id.ok_or(GroupError::GroupNotFound)?)
```